server/src/main/java/org/apache/druid/metadata/SQLMetadataRuleManager.java

```diff
@@ -236,7 +236,7 @@ public void poll()
   {
     try {
     
```

Comment:
Is this required for this change?

Reply:
This `ImmutableMap.copyOf(Map<? extends K, ? extends V> map)` is now overloaded with another method `ImmutableMap.copyOf(Iterable<? extends Entry<? extends K, ? extends V>> entries)`. Compiler was complaining when I tried to cast the params to `Map<String, List<Rule>>` hence created another variable with that type and used it to obtain ImmutableMap.